java_tools/gcc_map_reader/build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+defaultTasks 'jar'
+
+dependencies {
+    compile fileTree( dir: 'lib', includes: [ 'junit.jar', 'annotations.jar' ] )
+}
+
+sourceSets {
+    main {
+        java.outputDir = file( 'build/classes' )
```

Comment:
Why do we need non standard output folder? Please add comment into Gradle file if legit reason to make it obvious